src/pages/ContentfulPage.vue

```diff
@@ -111,13 +113,26 @@ const getPageFrameFromType = type => {
 	}
 };
 
+// The components that return kv-tailwind will get wrapped in that class,
```

Comment:
I think this will be a good way to migrate the contentful components over to use tailwinds